cmd/serve.go

```diff
@@ -179,18 +184,21 @@ func requireFlag(flagName string, flagValue string) {
 }
 
 func serve() error {
-	keypairFilename := "signing"
-	if err := token.GenerateKeypair(keypairFilename); err != nil {
-		glog.Errorf("Error generating key pair: %v", err)
+	if token.KeypairExists(keypairPrefix) {
+		glog.Infof("Using existing keypair in %v", keypairPrefix)
+	} else {
+		if err := token.GenerateKeypair(keypairPrefix); err != nil {
```

Comment:
i think we should generate keypair only on explicit request to generate them. else we should fail if the keypair don't exist

Reply:
can you please change it to throw error if keyvalue pair is not present